src/python/foglamp/translators/omf_translator.py

```diff
@@ -114,12 +87,13 @@
 # PI Server OMF reference - for detailed information
 # http://omf-docs.readthedocs.io/en/v1.0/Data_Msg_Sample.html#data-example
 
-_server_name = ""
+_event_loop = ""
+
 _relay_url = ""
 _producer_token = ""
 
 # The size of a block of readings to send in each transmission.
-_block_size = 50
+_block_size = 10
```

Comment:
the block size will be managed by the 'sending process' and it will manage the configuration, so this is a tmp solution.